.github/CODEOWNERS

```diff
@@ -34,6 +34,9 @@
 ## Snaps Team
 /packages/rate-limit-controller            @MetaMask/snaps-devs
 
+## Portfolio Team
+/packages/token-search-discovery-controller @MetaMask/portfolio
```

Comment:
Can you also place the `package.json` and `CHANGELOG.md` under joint ownership of our team and your team? So:

``` suggestion
/packages/token-search-discovery-controller/package.json      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
/packages/token-search-discovery-controller/CHANGELOG.md      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
```

Reply:
Oops, I accidentally made this a `suggestion` but I meant to suggest that the line I commented on should be kept, but that two extra lines should be added at the bottom of the file.